schema/home/fetch.js

```diff
@@ -22,6 +23,12 @@ export const featuredFair = () => {
   });
 };
 
+export const activeSaleArtworks = (accessToken) => {
+  return gravity.with(accessToken)('me/lot_standings').then((results) => {
```

Comment:
Following up from Slack discussion, "live" in this context means "sales that are active" and not LAI sales. I've pushed the change.